velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -1014,10 +1034,30 @@ void JoinFuzzer::verify(core::JoinType joinType) {
   const auto numKeys = nullAware ? 1 : randInt(1, 5);
 
   // Pick number and types of join keys.
-  const std::vector<TypePtr> keyTypes = generateJoinKeyTypes(numKeys);
+  std::vector<TypePtr> keyTypes = generateJoinKeyTypes(numKeys);
+  std::string semiFilter;
```

Comment:
It's better to avoid creating a new term, especially because the term "semi" could make people think about semi-join and cause confusion. What about we rename it as "nonKeyFilter" or just "extraFilter"?